jwst/datamodels/schemas/ifucube.schema.yaml

```diff
@@ -35,6 +35,10 @@ allOf:
           title: Weighting power for Modified Shepard Method
           type: number
           fits_keyword: WPOWER
+        channel:
```

Comment:
Can't the channel attribute (and corresponding keyword) that's already defined in core.schema.yaml be used for this, rather than duplicating it again here? The attribute is meta.instrument.channel. I notice that the current keyword comment field for that is "NIRCam channel: long or short", but that should be updated anyway, because the CHANNEL keyword is already used for both NIRCam and MIRI. We could update the comment to something more generic, like "Instrument channel."